### ISSUE TITLE
File corrupt?

### ISSUE THREAD

**Issue by YasNishida** [nsherry4/Peakaboo#270]
After installation, opening Peakaboo results in error, "Peakaboo is damaged and can't be opened".
version  5.5.0

MacBook Air 2019
OS 12.4 Monterey
intel core i5
<img width="283" alt="Screen Shot" src="https://user-images.githubusercontent.com/61447913/182011296-111aaacb-211c-476a-a7df-a553951e668a.png">

I have tried downloading 3 times and got the same error.

**Comment by nsherry4** [comment]
Thank you for filing a bug report.

This is an issue that I've been trying to solve for some time. I was able to resolve it by opening a terminal and running `xattr -rc /Applications/Peakaboo.app`, which (from what I understand) will **r**ecursively **c**lear extended attributes on the .app folder. After this I was able to launch the app normally.

Let me know if this works for you.

A more permanent fix is still required, so I'll leave this issue open until I have one.
